src/libraries/Microsoft.Extensions.DependencyInjection.Specification.Tests/src/Microsoft.Extensions.DependencyInjection.Specification.Tests.csproj

```diff
@@ -5,6 +5,7 @@
     <EnableDefaultItems>true</EnableDefaultItems>
     <CLSCompliant>false</CLSCompliant>
     <IsRuntimeAssembly>false</IsRuntimeAssembly>
+    <EnablePackageBaselineValidation>false</EnablePackageBaselineValidation>
```

Comment:
Presumably this is because it didn't have a 5.0.0 version, or...?  Needs a comment whenever we do this.